backend/src/api/PostController.ts

```diff
@@ -499,8 +499,15 @@ export default class PostController {
                 msg = 'OpenAI error';
             }
             this.logger.error(err);
-            // TODO analyze OpenAI response here and show custom error message once we will know how "run out of money" response looks like
-            return response.error('error', msg, 500);
+            try {
+                response.error('error', msg, 500);
+            } catch (err) {
+                // in case some chunks were already written
+                // should not happen now, just a precaution if the invariant in translateEntity is broken
+                this.logger.error('Error writing response');
+                this.logger.error(err);
+                response.end();
```

Comment:
We should still do `response.write('{"result":"error","code":"error"}')` - frontend looks for this message in every chunk and will popup an error in browser if it meets it
https://github.com/spaceshelter/orbitar/blob/ece8d877d0c216291dc1b262bc6f02d0d010f3fe/frontend/src/API/use/useInterpreter.ts#L71C39-L71C71

Reply:
I can add this, but it doesn't work as you expect:
<img width="446" alt="image" src="https://github.com/spaceshelter/orbitar/assets/2865203/a64b8d2e-66e4-43fb-a01c-ece34c6177ac">

Reply:
Ok, found how to make it work. Pushed.